litellm/model_prices_and_context_window_backup.json

```diff
@@ -1730,6 +1730,22 @@
         "supports_assistant_prefill": true,
         "supports_prompt_caching": true
     },
+    "claude-3-5-haiku-20241022": {
+        "max_tokens": 8192,
+        "max_input_tokens": 200000,
+        "max_output_tokens": 8192,
+        "input_cost_per_token": 0.000001,
+        "output_cost_per_token": 0.000005,
+        "cache_creation_input_token_cost": 0.00000125,
+        "cache_read_input_token_cost": 0.0000001,
+        "litellm_provider": "anthropic",
+        "mode": "chat",
+        "supports_function_calling": true,
+        "supports_vision": true,
```

Comment:
I think this is false.

![image](https://github.com/user-attachments/assets/e79c4c8a-5e72-4821-a7c5-3fe0aaff45c3)

and modifying the map locally, I get error from anthopic API

<img width="1139" alt="image" src="https://github.com/user-attachments/assets/559ed073-6aff-46a7-94c1-74c3bfda9117">